Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3395,8 +3395,22 @@ extension BrowserViewController: PreferencesObserver {
                 
                 // Clear ALL data when going from normal mode to private
                 // The other way around is handled in `removeTab`
-                let clearables: [Clearable] = [CookiesAndCacheClearable()]
-                _ = ClearPrivateDataTableViewController.clearPrivateData(clearables)
+                
+                let alert = UIAlertController(title: "Privacy", message: "Clearing session.. Please Wait..", preferredStyle: .alert)
```

Comment:
These will need to be localized. Also do we really need to do a popup? Seems a bit overkill 😅

Reply:
Hmm, we probably don't need the popup but I don't know of a better way to block the user from spamming the switch unlimited..

Reply:
What if we just disabled the switch until it finishes?